JS_L1/a1.js

```diff
@@ -0,0 +1,6 @@
+// Bài 1:
+// -	Input: Cho 1 object hoặc mảng
+// -	Output: kiểm tra xem object hoặc mảng đó rỗng không
+var input = {1:'cars'};
+var output = (Object.keys(input).length > 0) ? "is not an empty object" : "is an empty object"
```

Comment:
- Tại sao chỗ này không dùng string bình thường để in ra???
ví dụ: Object.keys(input).length > 0 ? `Input is not empty object` : 'Input is an empty object'
- Giải sử input là var input = undefined thì liệu Object.keys(input) => có thực hiện được không????